### PR TITLE
Task1

### PR DESCRIPTION
**Pull request by KareemZafan**
Edit two text files

### CODE REVIEW

File: test1.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-Hello Nardeen
```
Comment (EmanMustafa) [inline test1.txt:1]: Great

File: test2.txt [hunk 2]
```diff
@@ -1 +1,4 @@
-Hello World
```
Comment (EmanMustafa) [inline test2.txt:1]: mmm

**Review by EmanMustafa — CHANGES_REQUESTED** [review]

File: test1.txt [hunk 3]
```diff
@@ -1 +1,2 @@
...
\ No newline at end of file
+
```
Comment (Salma-NB) [inline test1.txt]: When pigs fly

File: test1.txt (hunk 1 above)
Comment (KareemZafan) [reply test1.txt:1]: Thanks,

Comment (KareemZafan) [reply test1.txt:1]: We kept Nardeen

File: test2.txt (hunk 2 above)
Comment (KareemZafan) [reply test2.txt:1]: Okay

**Review by EmanMustafa — APPROVED** [review]
tamam

**Pull request merged**